src/verinfast/agent.py

```diff
@@ -364,12 +370,13 @@ def parseRepo(self, path: str, repo_name: str):
                 with open(stats_input_file, 'w') as f:
                     f.write(json.dumps(filelist, indent=4))
 
+                template_definintion["filelist"] = filelist
                 # Calling modernmetric with subprocess works, but we might want to call
                 # Modernmetric directly, ala lines 91-110 from modernmetric main
                 with open(stats_output_file, 'w') as f:
                     with open(stats_error_file, 'w') as e:
                         subprocess.check_call(["modernmetric", f"--file={stats_input_file}"], stdout=f, stderr=e, encoding='utf-8')
-
+                template["stats"] = json.load(stats_output_file)
```

Comment:
You mean template_definition right?

Reply:
There's still an issue here when trying to add stats to the template definition. I'm not sure why `json.load(stats_output_file)` is loading an empty dictionary after `modernmetric` is run.